themes/custom/droopler_theme/scss/components/_d_p_reference_content.scss

```diff
@@ -24,15 +24,15 @@ $d-p-reference-content-content-padding: 5px !default;
   > .field--name-field-d-p-reference-content {
     padding-top: $d-p-reference-content-padding-top;
     padding-bottom: $d-p-reference-content-padding-bottom;
-    margin-bottom: 1.51rem;
-
+    margin-bottom: 0;
```

Comment:
skoro są paddingi na zmiennych to może warto wystawić to `0` też jako zmienna? @grzegorz-pietrzak-droptica 
zeby ktos mogl sobie zedytowac?

Reply:
Tak, możemy tak zrobić